src/main/java/com/vinyl/controller/UserController.java

```diff
@@ -15,14 +16,20 @@
 import com.vinyl.modelDTO.UserDTO;
 import com.vinyl.service.UserService;
 
+@Api(value="User Management", description="Operations pertaining to users in User Management System")
 @RestController
 public class UserController {
 
 	@Autowired
 	UserService userService;
 
+	@ApiOperation(value = "Create an account", response = ResponseEntity.class)
+	@ApiResponses(value = {
+			@ApiResponse(code = 201, message = "Successfully created user"),
```

Comment:
see if you can extract duplicate string as constants and use them in these annotations

Reply:
done